dask/dataframe/groupby.py

```diff
@@ -1449,7 +1449,16 @@ def _single_agg(
         if aggregate_kwargs is None:
             aggregate_kwargs = {}
 
-        columns = meta.name if is_series_like(meta) else meta.columns
+        if is_series_like(meta):
+            # in pandas 2.0, Series returned from value_counts have a name
+            # different from original object, but here, column name should
+            # still reflect the original object name
+            if func is _value_counts:
+                columns = self._meta.apply(pd.Series).name
```

Comment:
This `_value_counts` special casing feels a bit off. Is there a way we could move this logic into `_value_counts` itself?

Reply:
I know, ugly. I found a nicer way, see if this looks better.

Reply:
That looks great -- thanks for updating 👍